electron-app/src/components/dashboard/OverallBalances.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <v-card class="overall-balances-box mt-3 mb-6" :loading="isLoading">
+    <v-row no-gutters>
+      <v-col
+        cols="12"
+        md="4"
+        lg="4"
+        style="display: flex; flex-direction: column; justify-content: center;"
+      >
+        <div
+          class="rotkibeige display-3 pa-5 primary--text font-weight-bold"
+          style="height: 100%; width; 100%;"
+        >
+          net worth
+        </div>
+      </v-col>
+      <v-col cols="12" md="8" lg="8">
+        <div
+          class="display-3 pa-5"
+          style="display: flex; justify-content: center; align-items: center; height: 100%; width; 100%;"
+        >
+          <amount-display
+            show-currency="symbol"
+            fiat
+            :value="total"
+          ></amount-display>
+        </div>
+      </v-col>
+    </v-row>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { default as BigNumber } from 'bignumber.js';
+import { Component, Vue, Prop } from 'vue-property-decorator';
+import { createNamespacedHelpers } from 'vuex';
+
+import AmountDisplay from '@/components/display/AmountDisplay.vue';
+import { AssetBalance } from '@/model/blockchain-balances';
+import { Currency } from '@/model/currency';
+import { Zero } from '@/utils/bignumbers';
+
+const { mapGetters } = createNamespacedHelpers('session');
+const { mapGetters: mapBalanceGetters } = createNamespacedHelpers('balances');
+@Component({
+  components: { AmountDisplay },
+  computed: {
+    ...mapGetters(['currency']),
+    ...mapBalanceGetters(['aggregatedBalances'])
+  }
+})
+export default class OverallBox extends Vue {
+  @Prop({ required: false, default: false })
+  isLoading!: boolean;
+
+  currency!: Currency;
+  aggregatedBalances!: AssetBalance[];
+
+  get total(): BigNumber {
+    return this.aggregatedBalances.reduce(
```

Comment:
At least when using it on my data I get a big difference between the total net worth here and the TOTALS of the per assets balance. Should they not be the same?

Reply:
There was a bug here because I had not upgraded this component to use the updated `<amount-display>` properties as I had begin this PR before we fixed that. As a result, it was not converting from USD to your local currency. I will fixed this in the next commit, can you check again after the push?

If you're still getting big differences there something weird going on and we can investigate further.